fog/api/proto/ledger.proto

```diff
@@ -389,6 +389,6 @@ message TxOutResult {
 enum TxOutResultCode {
     NotFound = 0;
     Found = 1;
-    MalformedRequest = 2;
+    MalformedRequest = 2 [deprecated=true];
```

Comment:
This can no longer be returned, if a request is malformed we return a GRPC error. In practice this should never happen for well behaved clients.